spyder/app/mainwindow.py

```diff
@@ -2568,8 +2568,17 @@ def render_issue(self, description='', traceback=''):
     def report_issue(self, body=None, title=None, open_webpage=False):
         """Report a Spyder issue to github, generating body text if needed."""
         if body is None:
+            from spyder.plugins.console import (
+                    MAIN_BG_COLOR, MAIN_DEFAULT_FG_COLOR, MAIN_ERROR_FG_COLOR,
+                    MAIN_TB_FG_COLOR, MAIN_PROMPT_FG_COLOR)
             from spyder.widgets.reporterror import SpyderErrorDialog
-            report_dlg = SpyderErrorDialog(self, is_report=True)
+            report_dlg = SpyderErrorDialog(
+                    self, is_report=True,
+                    default_foreground_color=MAIN_DEFAULT_FG_COLOR,
+                    error_foreground_color=MAIN_ERROR_FG_COLOR,
+                    traceback_foreground_color=MAIN_TB_FG_COLOR,
+                    prompt_foreground_color=MAIN_PROMPT_FG_COLOR,
+                    background_color=MAIN_BG_COLOR)

```

Comment:
@dalthviz, I really don't like that you define and pass all these parameters to widgets that inherit or use `ConsoleBaseWidget`. I think there are two possible ways to simplify this:

1. Define these constants where `ConsoleBaseWidget` is defined and use them only there (perhaps by applying its `set_color_scheme` method on its `__init__`).
2. If that's not possible, define a single parameter called `is_dark_interface` and pass that parameter to widgets that inherit or use `ConsoleBaseWidget`

Reply:
And passing these parameters to `SpyderErrorDialog` is not needed either. Please restore this too.